packages/terra-site/src/examples/form-input/DefaultExample.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import FormInput from 'terra-form-input/lib/FormInput';
+
+class DefaultExample extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { input: 'Default Input' };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ input: event.target.value });
+  }
+
+  render() {
+    return(
+      <div>
+        <h4>Default</h4>
+        <FormInput
+          defaultValue="Default Input"
```

Comment:
I don't think this kind of input is correct react syntax actually. For an uncontrolled input, you just populate defaultValue while leaving onChange undefined. I'd make this one an uncontrolled input as well. You can read more about it [here](https://reactjs.org/docs/uncontrolled-components.html)

Reply:
I don't think the fact that this doesn't explicitly map the state back into the component via the `value` prop explicitly makes it invalid _React_ syntax. Admittedly, it is **not** a valid _controlled input_ example. As I want to keep the presentational part of this below (`<p>Input Provided: {this.state.input}</p>`), I'm going to make a clearer distinction between _controlled_ and _uncontrolled_ inputs by providing an example of both. 34572384